### PR TITLE
Mercury destination emblem

### PR DESCRIPTION
**Pull request by Trekoid**
- Replace Mercury Treasure Seeker with Fields of Glass
- Add missing Prophetic Arsenal
- Split public space emblems from achievement emblems
- Add Sonic Simulation to promotional section despite it being classified

### CODE REVIEW

File: src/setData/dlc1.js [hunk 1]
```diff
@@ -442,15 +442,21 @@ module.exports = [
...
-            title: 'Achievement Emblems',
+            title: 'Emblems',
+            items: [
+              1312472209, // Fields of Glass
+              2939572589, // Crossroads Emblem
+              4261480748, // Transit of Mercury
+            ],
+          },
+          {
+            title: 'Achievement Emblems - Curse of Osiris',
```
Comment (Jakosaur) [inline src/setData/dlc1.js]: Don't think the title needs " - Curse of Osiris" as Mercury is only available with Curse of Osiris and this section is on that page. Apart from that, very good changes. I'll close my PR as my changes have been put in here

**Comment by joshhunt** [comment]
> Add Sonic Simulation to promotional section despite it being classified

I'm a bit hesitant about adding in Classified items at this moment - there's nothing users can actually do with them on the page so its just going to annoy people, I feel. Maybe we can just comment it out until its unclassified?

File: src/setData/dlc1.js (hunk 1 above)
Comment (Trekoid) [reply src/setData/dlc1.js]: I was matching what was done on the base game page. I try to be consistent even if it is redundant, but I can see going for a streamlined version so I removed it from the title.

**Comment by Trekoid** [comment]
> I'm a bit hesitant about adding in Classified items at this moment - there's nothing users can actually do with them on the page so its just going to annoy people, I feel. Maybe we can just comment it out until its unclassified?

I have done the preorder and have the emblem, so it is obtainable. Here is what it looks like in Destiny Item Manager.
screen shot 2018-03-02 at 12 44 15 pm

There seems to be a filter in the Destiny Sets rendering that skips Classified items. Here is a screen shot of me running the branch locally on my laptop. The Sonic Simulation doesn't get rendered. Having it there uncommented, means it will automatically appear once the APIs are updated.
screen shot 2018-03-02 at 12 42 46 pm

**Comment by joshhunt** [comment]
Well there you go - you learn something new about the code you've written every day 😉

**Pull request merged**